services/src/test/java/org/exoplatform/addons/gamification/rest/TestChallengeRest.java

```diff
@@ -471,7 +471,7 @@ public void testDeleteChallenge() throws Exception {
               .key("startDate")
               .value(startDate)
               .key("endDate")
-              .value(endDate)
+              .value(Utils.toRFC3339Date(new Date(System.currentTimeMillis() - 2 * MILLIS_IN_A_DAY)))
```

Comment:
Why was this change needed to make the test pass ? What is the change that made this change fails (if so) ?

Reply:
this change made it fail here : https://github.com/Meeds-io/gamification/pull/402/files#diff-739c038ffda37355664bd909591a1624056a27d768cd2d4c5aed9e9e470aec4cR51
since deleting a challenge need the challenge to be ended